main.go

```diff
@@ -176,9 +177,26 @@ func mainWithExitCode() exitCode {
 		return failure("Failed to create container metadata provider: %v", err)
 	}
 
+	var extraSymbolEndpoints SymbolEndpoints
+	err = json.Unmarshal([]byte(args.extraSymbolEndpoints), &extraSymbolEndpoints)
+	if err != nil {
+		return failure("Unable to unmarshal extra symbol endpoints")
+	}
+
+	sites := extraSymbolEndpoints.getFieldString("Site")
+	apiKeys := extraSymbolEndpoints.getFieldString("ApiKey")
+	appKeys := extraSymbolEndpoints.getFieldString("AppKey")
+
+	sites = append(sites, args.site)
+	apiKeys = append(apiKeys, args.apiKey)
+	appKeys = append(appKeys, args.appKey)
+
 	var intakeURL string
 	apiKey := ""
 	if args.agentless {
+		if len(sites) > 1 {
+			return failure("Agentless mode does not handle more than one site")
+		}
```

Comment:
Since the environment variable is now explicitly used for additional symbol endpoints, as such it shouldn't impact reporting of profiles (only reporting of symbols), so we shouldn't need this check.